src/native/libs/System.Security.Cryptography.Native/opensslshim.c

```diff
@@ -127,6 +127,17 @@ static void OpenLibraryOnce(void)
     {
         DlOpen(MAKELIB("111"));
     }
+    // FreeBSD uses this for OpenSSL3 from ports. OpenSSL3.1 would be "13"
+    // Ports version of OpenSSL is used over base where possible
+    if (libssl == NULL)
+    {
+        DlOpen(MAKELIB("12"));
```

Comment:
Comment says `13`, code says `12`. Is that intentional?

Reply:
Yes and this confused me too. `openssl30` from ports uses `.12`. `openssl31` from ports uses `.13`. I left it at openssl3's `.12` as no other system or case listed seems to use openssl 3.1 nor have I had the time to test under openssl 3.1.

If desired, I can remove that part of the comment to reduce the confusion.

Reply:
I think adjusting the comment to say 3.0 is .12 is fine.